pkg/handler/nfts_test.go

```diff
@@ -0,0 +1,236 @@
+package handler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/defipod/mochi/pkg/config"
+	"github.com/defipod/mochi/pkg/entities"
+	"github.com/defipod/mochi/pkg/logger"
+	"github.com/defipod/mochi/pkg/repo/pg"
+	"github.com/defipod/mochi/pkg/response"
+	mock_indexer "github.com/defipod/mochi/pkg/service/indexer/mocks"
+	"github.com/defipod/mochi/pkg/util/testhelper"
+	"github.com/gin-gonic/gin"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHandler_GetNewListedNFTCollection(t *testing.T) {
+	cfg := config.LoadTestConfig()
+	db := testhelper.LoadTestDB("../../migrations/test_seed")
+	repo := pg.NewRepo(db)
+	log := logger.NewLogrusLogger()
+	s := pg.NewPostgresStore(&cfg)
+	entityMock := entities.New(cfg, log, repo, s, nil, nil, nil, nil, nil, nil, nil)
+
+	type fields struct {
```

Comment:
why don't use direct entities and log?

Reply:
we need a new `entity` to load in test config, repo, and service